persistence/jdbc/src/test/java/org/infinispan/persistence/jdbc/stringbased/JdbcStringBasedStoreTest.java

```diff
@@ -69,7 +69,9 @@ public void testNotCreateConnectionFactory() throws Exception {
             .dbMajorVersion(1)
             .dbMinorVersion(4);
 
-      storeBuilder.table().createOnStart(false);
+      storeBuilder.table()
+            .createOnStart(false)
+            .tableNamePrefix("mock_table_name");
```

Comment:
The `TableName` is initialized in the constructor and the `tableNamePrefix` is required.